cinema-soft/pom.xml

```diff
@@ -103,35 +124,8 @@
                     <target>9</target>
                 </configuration>
             </plugin>
-            <plugin>
```

Comment:
Does it not work with this plug in? If not, did you investigate why?

Reply:
It works.